server/server.cjs

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
+const mongoose = require('mongoose');
+const usersRouter = require('./api/users.cjs');
+require('dotenv').config();
+require('./config/config-passport.cjs');
+
+const app = express();
+const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short';
+
+app.use(morgan(formatsLogger));
+app.use(cors());
+app.use(express.json());
+
+app.use(express.static('public'));
+app.use('/users', usersRouter);
+
+app.use((_, res, __) => {
+  res.status(404).json({
+    status: 'error',
+    code: 404,
+    message: 'Connection error',
+    data: 'Not found',
+  });
+});
+
+app.use((error, _, res, __) => {
+  console.log(error.stack);
+  res.status(500).json({
+    status: 'Epic fail',
+    code: 500,
+    message: error.message,
+    data: 'Internal server error',
+  });
+});
+
+const PORT = process.env.PORT;
```

Comment:
warto się zastanowić, czy nie dać jeszcze alternatywy, np. || 3000

Reply:
Tu akurat w envie dałem port 3000 dlatego nie ma LUB 🙂

Reply:
Tak, rozumiem. Daje się jako zabezpieczenie, gdyby w dotenv coś nie zatrybiło :)

Reply:
Wydaje mi sie, że dodaje się na wszelki wypadek inny port niż zawarty w .env, jako backup - gdyby tamten był z jakichś powodów zajęty.

Reply:
To mam zmienić port w envie i dać lub 3000 czy zostawić 3000 w envie i dać w lub inny, a jeśli inny, to jaki?